app/processing/delta/get-outstanding-ledger-values.js

```diff
@@ -0,0 +1,26 @@
+const { AP, AR } = require('../../ledgers')
+
+const getOutstandingLedgerValues = (previousPaymentRequests) => {
```

Comment:
I've opted to rename references to `unsettled` as it's no longer appropriate given we now know that we don't need to worry about the settlement status of AR.  We just need to know what debt is outstanding.  Unsettled values is specific to AP.

So I've chosen `getOutstandingLedgerValues` as essentially that's what we need to find out.  ie, how much is outstanding on each ledger that we should be looking to cancel out.